hedera-node/src/main/java/com/hedera/services/fees/calculation/contract/queries/ContractCallLocalResourceUsage.java

```diff
@@ -96,7 +112,46 @@ private FeeData usageFor(Query query, StateView view, ResponseType type, Optiona
 			if (queryCtx.isEmpty()) {
 				response = dummyResponse(op.getContractID());
 			} else {
-				response = delegate.perform(op, Instant.now().getEpochSecond());
+				TransactionBody body =
+						SignedTxnAccessor.uncheckedFrom(op.getHeader().getPayment()).getTxn();
+				final var senderId = Id.fromGrpcAccount(body.getTransactionID().getAccountID());
+				final var contractId = Id.fromGrpcContract(op.getContractID());
+
+				/* --- Load the model objects --- */
+				final var sender = accountStore.loadAccount(senderId);
+				final var receiver = accountStore.loadContract(contractId);
```

Comment:
With the refactored code, loading accounts will throw exception if `validateUsable` fails. This means that once it throws it will go to the catch and will throw with `throw new IllegalStateException(internal);`.

In the legacy implementation if f.e `validateContractExistence` fails, the response code is checked and we are not throwing with `throw new IllegalStateException(internal);` but rather:
```Java
ResponseHeader responseHeader = RequestBuilder.getResponseHeader(callResponseStatus, 0l,
					ANSWER_ONLY, ByteString.EMPTY);
			responseToReturn = ContractCallLocalResponse.newBuilder().setHeader(responseHeader).build();
			if (log.isDebugEnabled()) {
				log.debug("contractCallLocal  -Invalid Contract ID "
						+ TextFormat.shortDebugString(transactionContractCallLocal.getContractID()));
			}
```